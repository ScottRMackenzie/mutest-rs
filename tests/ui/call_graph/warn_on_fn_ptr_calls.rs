//@ print-targets
//@ stdout
//@ stderr

fn impl_call() {}

fn make_fn_ptr_call(f: fn()) {
    f();
}

#[test]
fn test() {
    make_fn_ptr_call(impl_call);
}
