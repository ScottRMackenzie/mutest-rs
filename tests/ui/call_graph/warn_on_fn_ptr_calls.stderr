warning: encountered dynamic call during call graph construction
 --> tests/ui/call_graph/warn_on_fn_ptr_calls.rs:8:5
  |
8 |     f();
  |     ^^^ call to fn()
  |
  = note: in make_fn_ptr_call

warning: 1 warning emitted

